cmd/validator/flags/flags.go

```diff
@@ -109,25 +104,27 @@ var (
 		Usage: `Comma separated list of key value pairs to pass as gRPC headers for all gRPC calls.
 		Example: --grpc-headers=key=value`,
 	}
-	// GRPCGatewayHost specifies a gRPC gateway host for the validator client.
-	GRPCGatewayHost = &cli.StringFlag{
-		Name:  "grpc-gateway-host",
-		Usage: "Host on which the gateway server runs on.",
-		Value: DefaultGatewayHost,
-	}
-	// GRPCGatewayPort enables a gRPC gateway to be exposed for the validator client.
-	GRPCGatewayPort = &cli.IntFlag{
-		Name:  "grpc-gateway-port",
-		Usage: "Enables gRPC gateway for JSON requests.",
-		Value: 7500,
-	}
-	// GRPCGatewayCorsDomain serves preflight requests when serving gRPC JSON gateway.
-	GRPCGatewayCorsDomain = &cli.StringFlag{
-		Name: "grpc-gateway-corsdomain",
-		Usage: `Comma separated list of domains from which to accept cross origin requests (browser enforced).
-		This flag has no effect if not used with --grpc-gateway-port.
-`,
-		Value: "http://localhost:7500,http://127.0.0.1:7500,http://0.0.0.0:7500,http://localhost:4242,http://127.0.0.1:4242,http://localhost:4200,http://0.0.0.0:4242,http://127.0.0.1:4200,http://0.0.0.0:4200,http://localhost:3000,http://0.0.0.0:3000,http://127.0.0.1:3000"}
+	// HTTPServerHost specifies a HTTP server host for the validator client.
+	HTTPServerHost = &cli.StringFlag{
+		Name:    "http-host",
+		Usage:   "Host on which the HTTP server runs on.",
+		Value:   DefaultHTTPServerHost,
+		Aliases: []string{"grpc-gateway-host"},
+	}
+	// HTTPServerPort enables a HTTP server port to be exposed for the validator client.
+	HTTPServerPort = &cli.IntFlag{
+		Name:    "http-port",
+		Usage:   "Port on which the HTTP server runs on.",
+		Value:   7500,
+		Aliases: []string{"grpc-gateway-port"},
+	}
+	// HTTPServerCorsDomain adds accepted cross origin request addresses.
+	HTTPServerCorsDomain = &cli.StringFlag{
+		Name:    "corsdomain",
+		Usage:   `Comma separated list of domains from which to accept cross origin requests (browser enforced).`,
+		Value:   "http://localhost:7500,http://127.0.0.1:7500,http://0.0.0.0:7500,http://localhost:4242,http://127.0.0.1:4242,http://localhost:4200,http://0.0.0.0:4242,http://127.0.0.1:4200,http://0.0.0.0:4200,http://localhost:3000,http://0.0.0.0:3000,http://127.0.0.1:3000",
```

Comment:
Same for this? It should improve readability ever so slightly

Reply:
like name each of these urls?

Reply:
Think it would be okay to just have a cont for the entire string, at least that way it's not in the middle of the code but can be modified from the top. Making them each individual values would be more of a pain:

`const DefaultHTTPCorsDomains = "http://localhost:7500,http://127.0.0.1:7500,http://0.0.0.0:7500,http://localhost:4242,http://127.0.0.1:4242,http://localhost:4200,http://0.0.0.0:4242,http://127.0.0.1:4200,http://0.0.0.0:4200,http://localhost:3000,http://0.0.0.0:3000,http://127.0.0.1:3000"`